src/components/Alert/Alert.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const fromTheme = (component, field) => props => props.theme.atoms[component][props.type][field];
+
+const StyledAlert = styled.div`
+  background: ${fromTheme('Alert', 'background')};
+  border-radius: 4px;
+  border: 2px solid ${fromTheme('Alert', 'borderColor')};
+  color: ${fromTheme('Alert', 'color')};
+  margin: 8px 0 8px 0;
+  min-height: 1em;
+  opacity: ${props => (props.visible ? '1' : '0')};
+  padding: 1em;
+  transition: opacity 0.2s linear;
+
+  .fa-remove {
```

Comment:
it feels like we should avoid nesting and use:

```
const CloseIcon = styled.i`
  float: right;
  cursor: pointer;
  margin-left: 1em;
`
```

```
<CloseIcon onClick={onClose} className="fa fa-remove" />
```

but I'm unsure about conventions.